src/urh/controller/CompareFrameController.py

```diff
@@ -864,6 +864,11 @@ def save_protocol(self):
 
         if filename.endswith(".bin"):
             self.proto_analyzer.to_binary(filename, use_decoded=True)
+        elif filename.endswith(".pcapng"):
+            data_link_type, ok = QInputDialog.getInt(self, "Link type", 
+                "Interface Link Type to use (must probably one between DLT_USER0-DLT_USER15 (147-162)):", 147)
+            if(ok):
```

Comment:
Braces are not needed:
```suggestion
            if ok:
```